Utilities/CookieCutter/{{cookiecutter.example_name}}/Code.cxx

```diff
@@ -37,30 +38,30 @@ int main( int argc, char* argv[] )
   constexpr unsigned int Dimension = 2;
 
   using PixelType = float;
-  using ImageType = itk::Image< PixelType, Dimension >;
+  using ImageType = itk::Image<PixelType, Dimension>;
 
-  using ReaderType = itk::ImageFileReader< ImageType >;
+  using ReaderType = itk::ImageFileReader<ImageType>;
   ReaderType::Pointer reader = ReaderType::New();
-  reader->SetFileName( inputFileName );
+  reader->SetFileName(inputFileName);
 
-  using FilterType = itk::{{ cookiecutter.class_name }}< ImageType, ImageType >;
+  using FilterType = itk::{ { cookiecutter.class_name } }<ImageType, ImageType>;
```

Comment:
The templates brackets should be `{{`

Reply:
Fixed and protected with clang-format off/on